versions.props

```diff
@@ -1,5 +1,5 @@
 org.slf4j:* = 1.7.36
-org.apache.avro:avro = 1.10.2
+org.apache.avro:avro = 1.11.1
```

Comment:
Of note, Spark's current 3.3 branch uses Avro 1.11.0.

We are mostly independent of them, but good to be aware of.